### PR TITLE
FOGL-3751 allow UTC timestamp by default for http south

### PR DESCRIPTION
**Pull request by praveen-garg**

### CODE REVIEW

File: python/fledge/plugins/south/http_south/http_south.py [hunk 1]
```diff
@@ -298,11 +309,24 @@ def __init__(self, config):
...
             for payload in payload_block:
                 asset = "{}{}".format(self.config_data['assetNamePrefix']['value'], payload['asset'])
-                timestamp = payload['timestamp']
-
-                # HOTFIX: To ingest readings sent from fledge sending process
-                if not timestamp.rfind("+") == -1:
-                    timestamp = timestamp + ":00"
+                dt_str = payload['timestamp']
+
+                if self.config_data['readingsTimestampTimezone']['value'] == 'UTC':
```
Comment (MarkRiddoch) [inline python/fledge/plugins/south/http_south/http_south.py]: This should only be done if there is no timezone in the incoming timestamp.

Comment (praveen-garg) [reply python/fledge/plugins/south/http_south/http_south.py]: So  when north will send `%Y-%m-%d %H:%M:%S.%f+00:00"` and sending process will send in `"%Y-%m-%dT%H:%M:%S.%fZ"`, I think  we can simply check that if timestamp string ends with Z `zulu time` then do the conversion else keep as it is.

correct? Or do we need more checking around +00:00 etc.

Comment (ashwinscale) [reply python/fledge/plugins/south/http_south/http_south.py]: Yes, currently we need to use  :
                    fmt = "%Y-%m-%dT%H:%M:%S.%fZ"

We also have an additional dependency on tzlocal.

**Comment by praveen-garg** [comment]
@ashwinscale can you please verify this?

File: python/fledge/plugins/south/http_south/http_south.py (hunk 1 above)
Comment (praveen-garg) [reply python/fledge/plugins/south/http_south/http_south.py]: Resolved!

**Pull request merged**